toqito/matrix_props/is_stochastic.py

```diff
@@ -0,0 +1,71 @@
+"""Check is a matrix is stochastic."""
+
+import numpy as np
+
+from toqito.matrix_props import is_nonnegative, is_square
+
+
+def is_stochastic(mat: np.ndarray, mat_type: str) -> bool:
```

Comment:
Nice job on the consolidation. Honestly, we can probably do the same with `is_stochastic` and `is_doubly_stochastic` as well. I think we could just use `"right"`, `"left"`, and `"doubly"` as the `mat_type` arguments.